src/Css.elm

```diff
@@ -3978,6 +3987,110 @@ backgroundPosition4 (Value horiz) (Value horizAmount) (Value vert) (Value vertAm
 
 
 
+-- BACKGROUND REPEAT --
+
+
+{-| Sets [`background-repeat`](https://css-tricks.com/almanac/properties/b/background-repeat/)
+
+    backgroundRepeat repeat
+
+    backgroundRepeat repeatX
+
+If you need to set horizontal and vertical direction separately, see
+[`backgroundRepeat2`](#backgroundRepeat2)
+
+-}
+backgroundRepeat :
+    Value
+        { repeat : Supported
+        , repeatX : Supported
+        , repeatY : Supported
+        , space : Supported
+        , round : Supported
+        , initial : Supported
+        , unset : Supported
+        , inherit : Supported
+        }
+    -> Style
+backgroundRepeat (Value repeat) =
+    AppendProperty ("background-repeat:" ++ repeat)
+
+
+{-| Sets [`background-repeat`](https://css-tricks.com/almanac/properties/b/background-repeat/) along the horizontal axis, then the vertical axis.
+
+    backgroundRepeat2 repeat space
+
+    backgroundRepeat2 space round
+
+If you only need to set one value for both, see
+[`backgroundRepeat`](#backgroundRepeat) instead.
```

Comment:
Love the explanation and linking between `backgroundRepeat` and `backgroundRepeat2`! 😍